spec/features/facets_spec.rb

```diff
@@ -1,16 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails_helper'
-
-RSpec.describe 'Facets', type: :feature do
-  context 'when viewing the Catalog page' do
-    before do
-      visit search_catalog_path
-    end
-
-    it 'the Access facet is first in the list and expanded' do
```

Comment:
@marlo-longley it might be worth moving the test that the facet is first over to your `spec/features/digital_content_filter_spec.rb` test. I think that'd maintain the decision made in #442.

Reply:
@taylor-steve thank you -- added back.